models.go

```diff
@@ -16,255 +16,131 @@ type Plex struct {
 }
 
 // Metadata ...
 type Metadata struct {
-	Media                 []Media `json:"Media"`
-	Player                Player  `json:"Player"`
-	Session               Session `json:"Session"`
-	User                  User    `json:"User"`
-	AddedAt               string  `json:"addedAt"`
-	Art                   string  `json:"art"`
-	ContentRating         string  `json:"contentRating"`
-	Duration              string  `json:"duration"`
-	GrandparentArt        string  `json:"grandparentArt"`
-	GrandparentKey        string  `json:"grandparentKey"`
-	GrandparentRatingKey  string  `json:"grandparentRatingKey"`
-	GrandparentTheme      string  `json:"grandparentTheme"`
-	GrandparentThumb      string  `json:"grandparentThumb"`
-	GrandparentTitle      string  `json:"grandparentTitle"`
-	GUID                  string  `json:"guid"`
-	Index                 string  `json:"index"`
-	Key                   string  `json:"key"`
-	LastViewedAt          string  `json:"lastViewedAt"`
-	LibrarySectionID      string  `json:"librarySectionID"`
-	LibrarySectionKey     string  `json:"librarySectionKey"`
-	LibrarySectionTitle   string  `json:"librarySectionTitle"`
-	OriginallyAvailableAt string  `json:"originallyAvailableAt"`
-	ParentIndex           string  `json:"parentIndex"`
-	ParentKey             string  `json:"parentKey"`
-	ParentRatingKey       string  `json:"parentRatingKey"`
-	ParentThumb           string  `json:"parentThumb"`
-	ParentTitle           string  `json:"parentTitle"`
-	RatingCount           string  `json:"ratingCount"`
-	Rating                float64 `json:"rating"`
-	RatingKey             string  `json:"ratingKey"`
-	SessionKey            string  `json:"sessionKey"`
-	Summary               string  `json:"summary"`
-	Thumb                 string  `json:"thumb"`
-	Title                 string  `json:"title"`
-	TitleSort             string  `json:"titleSort"`
-	Type                  string  `json:"type"`
-	UpdatedAt             string  `json:"updatedAt"`
-	ViewCount             string  `json:"viewCount"`
-	ViewOffset            string  `json:"viewOffset"`
-	Year                  string  `json:"year"`
+	Player                Player       `json:"Player"`
+	Session               Session      `json:"Session"`
+	User                  User         `json:"User"`
+	AddedAt               int          `json:"addedAt"`
+	Art                   string       `json:"art"`
+	ContentRating         string       `json:"contentRating"`
+	Duration              int          `json:"duration"`
+	GrandparentArt        string       `json:"grandparentArt"`
+	GrandparentKey        string       `json:"grandparentKey"`
+	GrandparentRatingKey  string       `json:"grandparentRatingKey"`
+	GrandparentTheme      string       `json:"grandparentTheme"`
+	GrandparentThumb      string       `json:"grandparentThumb"`
+	GrandparentTitle      string       `json:"grandparentTitle"`
+	GUID                  string       `json:"guid"`
+	Index                 int64        `json:"index"`
```

Comment:
Awesome!

Everything seems to work except when calling `GetSessions()`, Plex returns json with the `Index` field as a string and Go complains about this. 

This is obviously minor, but how would you solve this?

Reply:
Ah I think I never tested it with a running session. I overwrote added the fields to the V1 struct to over write it. Quick two line change.